pkg/apis/extensions/v1beta1/conversion_generated.go

```diff
@@ -1866,6 +1866,7 @@ func autoConvert_v1beta1_PodSecurityPolicySpec_To_extensions_PodSecurityPolicySp
 		return err
 	}
 	out.ReadOnlyRootFilesystem = in.ReadOnlyRootFilesystem
+	out.SeccompProfiles = in.SeccompProfiles
```

Comment:
I really think this should be 1 policy for the first implementation. Or we need to make it very clear things will only get more restricted not less. We made the distinct decision in docker that 1 profile holds the ultimate source of truth to avoid this confusion.